ui/packages/consul-ui/app/services/repository/hcp-link.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import RepositoryService from 'consul-ui/services/repository';
+import dataSource from 'consul-ui/decorators/data-source';
+
+export default class HcpLinkService extends RepositoryService {
+  @dataSource('/:partition/:ns/:dc/hcp-link')
+  async fetch({ partition, ns, dc }, { uri }, request) {
+    let result;
+    try {
+      result = (
+        await request`
+      GET /api/hcp/v2/link/global
+    `
+      )((headers, body) => {
+        return {
+          meta: {
+            version: 2,
+            uri: uri,
+          },
```

Comment:
Is this just "magic" ™️  to make the repo service happy?

Reply:
yes 😅